scripts/populate_db.py

```diff
@@ -28,7 +28,7 @@ def run_db_stuff(create_schema, recreate_db, populate_data, drop_db):
     # Create schema.
     if create_schema:
         engine = create_engine(CorporaDbConfig().database_uri)
-        if not database_exists(engine.url):
+        if not database_exists(engine.url):  # This never works because the url does not include the database name
```

Comment:
Can you elaborate a little on that comment?

Reply:
I took out the comment and filed a bug https://github.com/chanzuckerberg/corpora-data-portal/issues/870